src/call_builder.js

```diff
@@ -73,7 +73,7 @@ export class CallBuilder {
       timeout = setTimeout(() => {
         es.close();
         es = createEventSource();
-      }, 15*1000);
+      }, options.every || 15*1000);
```

Comment:
Let's change the name of this parameter to `reconnectTimeout` and update jsdoc.

Reply:
Updated.